manifest.php

```diff
@@ -43,11 +43,11 @@
     'name'        => 'taoQtiItem',
     'label'       => 'QTI item model',
     'license'     => 'GPL-2.0',
-    'version'     => '25.4.2',
+    'version'     => '25.4.3',
```

Comment:
I believe we should bmp at least the minor version here (in fact, I would even consider bumping a major one), the PR introduces a major dependency update, which on its own may be a breaking change.